src/edit/DistortableCollection.Edit.js

```diff
@@ -303,28 +303,31 @@ L.DistortableCollection.Edit = L.Handler.extend({
         statusUrl = opts.statusUrl + data;
         // repeatedly fetch the status.json
         self.updateInterval = setInterval(function intervalUpdater() {
-          $.ajax(statusUrl + '?' + Date.now(), {// bust cache with timestamp
-            type: 'GET',
-            crossDomain: true,
-          }).done(function(data) {
-            opts.updater(data);
-          });
+          var request = new XMLHttpRequest();
+          request.onload = function(event) {
```

Comment:
We can just remove the event parameter if we don't use it
```suggestion
          request.onload = function() {
```